taxes.md

```diff
@@ -26,6 +26,8 @@ You'll see sales tax on your invoice if you live in:
 * Washington
 * West Virginia
 
+In order to collect sales tax correctly, we need your full billing address. If your account does not have a full billing address on record, we may apply an extra charge to your invoices based on the average sales tax rate of municipalities that also charge sales tax on transactions without known sales origins.
```

Comment:
"May apply an extra charge" sounds like a penalty fee. Is this a sales tax calculation or are we increasing the price to account for unknown tax? If it is an estimated tax, that contrasts with the first sentence re. collecting sales tax correctly by suggesting that this is an incorrect/wrong/non-legit collection.

Does this meaning compare?
> If we can't determine sales tax due because we don't have your billing address on record, we collect the maximum/average/Chicago/etc sales tax, currently X%.

Reply:
It would essentially be a penalty fee. We do not collect sales tax on any payments that are not tied to a full billing address but we are liable for and are currently paying some state and local governments extra taxes on a portion of all sales without known addresses. 

Does this wording make it more clear? Earlier in the policy, we state that the rate being collected is determined by your billing address so this section would just explain what might happen if we are missing full billing addresses.

> If your account does not have a full billing address on record, we may apply an extra charge to your future invoices. That charge would be a percentage of your subscription price, based on the average sales tax rate of areas which charge sales tax on transactions without known address.